packages/devextreme/testing/tests/DevExpress.ui.widgets/listParts/commonTests.js

```diff
@@ -3683,7 +3726,7 @@ QUnit.module('keyboard navigation', {
 
         keyboard.keyDown('up');
         this.clock.tick(10);
-        assert.ok($selectAllItem.hasClass('dx-state-focused'), 'selectAll checkbox is focused');
+        assert.ok($selectAllItem.hasClass(`${LIST_FOCUS_STATE_CLASS}`), 'selectAll checkbox is focused');
```

Comment:
string literal is excess here and below
```suggestion
        assert.ok($selectAllItem.hasClass(LIST_FOCUS_STATE_CLASS), 'selectAll checkbox is focused');
```

Reply:
[c0a7c41](https://github.com/DevExpress/DevExtreme/pull/27230/commits/c0a7c4123abcac5a3446f5d8e33cde420436af34)